docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,313 @@
+Each pod is configured following the `spec` field in the deployment.
+Given that it can exist a lot of Traefik instances, a piece is required to forward the traffic
+to any of the instance: a [`Service`](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#Service).
+This process is called load balancing.
+Create a file called `02-traefik-services.yml` and insert the two `Service` resources:
+
+```yaml tab="02-traefik-services.yml"
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-dashboard-service
+
+spec:
+  type: LoadBalancer
+  ports:
+    - port: 8080
+      targetPort: dashboard
+  selector:
+    app: traefik
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-web-service
+
+spec:
+  type: LoadBalancer
+  ports:
+    - targetPort: web
+      port: 80
+  selector:
+    app: traefik
+```
+
+!!! warning
+    It is possible to expose a service in different manners. Depending on your working environment and use case,
+    the `spec.type` might change. It is strongly recommended understanding the available [service types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types).
+
+Traefik has an account and permissions.
+Traefik has a deployment.
+Traefik can be exposed outside the cluster.
+Traefik is ready to start.
+
+Apply the files on your cluster to start Traefik.
+
+```shell
+kubectl apply -f 00-role.yml \
+              -f 00-account.yml \
+              -f 01-role-binding.yml \
+              -f 02-traefik.yml \
+              -f 02-traefik-services.yml
+```
+
+## Proxying applications
+
+The missing part, now, is the business application behind the reverse-proxy.
+For this guide, [traefik/whoami](https://github.com/traefik/whoami) is used,
+but the concepts are equivalent for any other application.
+
+Whoami is a simple HTTP server running on the port 80 which answers host related information
+to the incoming requests. As usual, start by creating a file called `03-whoami.yml` and
+paste the following `Deployment` resource:
+
+```yaml tab="03-whoami.yml"
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: whoami
+  labels:
+    app: whoami
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: whoami
+  template:
+    metadata:
+      labels:
+        app: whoami
+    spec:
+      containers:
+        - name: whoami
+          image: traefik/whoami
+          ports:
+            - name: web
+              containerPort: 80
+```
+
+And continue by creating the following `Service` resource in a file called `03-whoami-services.yml`:
+
+```yaml tab="03-whoami-services.yml"
+apiVersion: v1
+kind: Service
+metadata:
+  name: whoami
+
+spec:
+  ports:
+    - name: web
+      port: 80
+      targetPort: web
+      
+  selector:
+    app: whoami
+```
+
+Traefik is an ingress controller.
+An ingress controller is a software that understands Ingress resources and adapts to them.
```

Comment:
Now it's expands on the concept of an ingress controller that was left in the air previously. If we clarify this there, on the beginning, I believe this section can be removed.